src/arkode/arkode_lsrkstep.c

```diff
@@ -419,11 +421,7 @@ int lsrkStep_FullRHS(ARKodeMem ark_mem, sunrealtype t, N_Vector y, N_Vector f,
     break;
 
   case ARK_FULLRHS_END:
-    /* No further action is needed if STS since the currently
-	    available STS methods evaluate the RHS at the end of each time step
-      If the stepper is an SSP, fn is updated and reused at the beginning 
-      of the step unless ark_mem->fn_is_current is changed by ARKODE.
-      */
+    /* No further action is needed if STS since the currently available STS methods evaluate the RHS at the end of each time step. If the stepper is an SSP, fn is updated and reused at the beginning of the step unless ark_mem->fn_is_current is changed by ARKODE. */
```

Comment:
Let's break up this overly long line:
```suggestion
    /* No further action is needed if STS since the currently available STS methods 
       evaluate the RHS at the end of each time step. If the stepper is an SSP, fn is 
       updated and reused at the beginning of the step unless 
       ark_mem->fn_is_current is changed by ARKODE. */
```